core/src/settings.rs

```diff
@@ -346,6 +346,8 @@ pub struct PropagatedState {
     pub(crate) tractor_requirements: TractorRequirements,
     #[serde(default)]
     pub(crate) max_rank: MaxRank,
+    #[serde(default)]
+    pub(crate) autodraw_speed_ms: u32,
```

Comment:
minor nit (don't actually make this change):

Ideally this would actually default to `None`, so you can identify the case where the `autodraw_speed_ms` is not set.

However, there's no stateful database, and the number of players drops to zero overnight, so we can probably ignore that (and also drop a bunch of the serde(default) tags)